airbyte_cdk/sources/streams/concurrent/partitions/partition.py

```diff
@@ -40,21 +40,6 @@ def stream_name(self) -> str:
         """
         pass
 
-    @abstractmethod
-    def close(self) -> None:
-        """
-        Closes the partition.
-        """
-        pass
-
-    @abstractmethod
-    def is_closed(self) -> bool:
```

Comment:
i did a very quick search of our cdk repo and never saw an usage of this method. What was the original intention of having this method?